lib/posthog/application.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Posthog.Application do
+  @moduledoc false
+
+  use Application
+
+  def start(_type, _args) do
+    children = [
+      # Start Cachex for feature flag event deduplication.
+      # The 50,000 entries limit is the same used for posthog-python, but otherwise arbitrary.
+      {Cachex, name: :posthog_feature_flag_cache, limit: 50_000, policy: Cachex.Policy.LRW}
```

Comment:
Also, should be LRU rather than LRW because you're not writing to the cache when you match it

Reply:
Yeah, I meant it to be LRU. Typo.